modules/core/src/lib/deck.js

```diff
@@ -559,8 +565,15 @@ export default class Deck {
           _pickRequest
         )
       );
+      const shouldGenerateInfo = _pickRequest.callback || this.props.getTooltip;
+      let pickedInfo;
+      if (shouldGenerateInfo) {
+        pickedInfo = result.find(info => info.index >= 0) || emptyInfo;
+      }
+      if (this.props.getTooltip) {
+        this.tooltip.setTooltip(this.props.getTooltip, pickedInfo);
```

Comment:
Just call `getTooltip` here - the `Tooltip` class does not have to know the user-facing interface.